### PR TITLE
Read habitatdata

### PR DESCRIPTION
**Pull request by ToonHub**
Add function to read habitatmap

### CODE REVIEW

File: R/read_habitatdata.R [hunk 1]
```diff
@@ -150,3 +152,215 @@ read_habitatmap_stdized <-
...
+        return(result)
+
+    }
+
+
+
+#' Return the data source \code{habitatmap}
+#'
+#' \code{read_habitatmap} returns all polygons of the \code{habitatmap} (De Saeger et al., 2018) that (partially)
+#' contain habitat or a Regional Important Biotope (RIB).
```
Comment (florisvdh) [inline R/read_habitatdata.R]: last line to be skipped

**Comment by ToonHub** [comment]
I changed variable names
I added an extra parameter select_hab. If TRUE only hab/rib is selected. Default value is FALSE.

**Comment by ToonHub** [comment]
I converted most string variables to characters

st_read(..., stringsAsFactors = FALSE) didn't seem to work,.. so I converted the factors to characters after reading the shapefile

**Comment by hansvancalster** [comment]
You can use `read_sf` instead if `st_read` to have stringsAsFactors = FALSE and return a tibble by default.

**Comment by florisvdh** [comment]
Great additions @ToonHub, thanks!
I suggest to merge!

**Pull request merged**